src/App.tsx

```diff
@@ -8,11 +8,30 @@ export const tabs: Tab[] = [
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 
-const App: React.FC = () => (
-  <div className="App">
-    <h1>Selected tab is&nbsp;Tab 1</h1>
-    <div>{tabs.length}</div>
-  </div>
-);
+const App: React.FC = () => {
+  const [selectedTabId, setSelectedTabId] = useState<string>(tabs[0].id);
+
+  const onTabSelected = (id: string) => {
+    setSelectedTabId(id);
+  };
+
+  const tabList = [...tabs];
+
+  return (
+    <div className="App">
+      <h1>
+        Selected tab is&nbsp;
+        {tabList.find(tab => tab.id === selectedTabId)?.title}
```

Comment:
Will be better to store the result of find method in some variable and use this variable in jsx